frontend/app/events/eventDetail/eventDetail.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute, Params } from "@angular/router";
+
+import { Event, EventService } from "./../";
+
+@Component({
+    moduleId: module.id,
+    selector: "cg-event-detail-page",
+    templateUrl: "eventDetail.component.html",
+})
+
+export class EventDetailComponent implements OnInit {
+    private activeEvent: Event;
+    private errorMessage: string;
+    constructor(private eventService: EventService, private route: ActivatedRoute) {
+        this.activeEvent = new Event();
```

Comment:
Hm das find ich auch nicht so schön. Was sagt ` Event()` aus? Typescript 2 hat nullable Typen. https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#null--and-undefined-aware-types